Makefile

```diff
@@ -83,10 +105,13 @@ bin/goreleaser:
 	@mkdir -p bin
 	@mkdir -p tmpgoreleaser
 	curl -sfL https://goreleaser.com/static/run | VERSION=v${GORELEASER_VERSION} TMPDIR=${PWD}/tmpgoreleaser bash -s -- --version
-	mv tmpgoreleaser/goreleaser bin/
+	@mv tmpgoreleaser/goreleaser bin/
 	@rm -rf tmpgoreleaser
 
-.PHONY: help
-.DEFAULT_GOAL := help
-help:
-	@grep -h -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-10s\033[0m %s\n", $$1, $$2}'
+bin/bats:
+	@mkdir -p bin
+	@mkdir -p tmpbats
+	git clone https://github.com/bats-core/bats-core.git tmpbats
+	@cd tmpbats && bash ./install.sh ${PWD}/bin
+	@mv bin/bin/bats bin/ && rm -rf bin/bin
+	@rm -rf tmpbats
```

Comment:
binary is not installed properly. the goreleaser creates an issue when testing so I had to disable it in makefile (we can address goreleaser issue in another PR). could you make sure that the binary is available as `secret-init/bin/bats`. 

You can verify if it works by running the commands below from the secret init path (you will have to comment out goreleaser as it will fail on dependency install)

```bash
$ export PATH="$PATH:$(pwd)/bin"
$ make deps
$ make test-e2e
```